gpflow/models/gplvm.py

```diff
@@ -262,5 +267,6 @@ def predict_f(
             var = tf.tile(tf.expand_dims(var, 1), shape)
         return mean + self.mean_function(Xnew), var
 
-    def predict_log_density(self, data: OutputData) -> tf.Tensor:
+    # type-ignore is because of changed method signature:
+    def predict_log_density(self, data: OutputData) -> tf.Tensor:  # type: ignore
```

Comment:
Why does this even change the abstract signature, given that it's not implemented?!

Reply:
> Why does this even change the abstract signature, given that it's not implemented?!

You're right. I'll update it to have the correct signature instead.